package.json

```diff
@@ -101,7 +102,11 @@
     "now": "^12.1.8",
     "npm-run-all": "4.1.5",
     "octicons": "^6.0.1",
+    "postcss": "7.0.14",
+    "postcss-load-config": "2.0.0",
     "postcss-loader": "^2.0.6",
+    "postcss-node-sass": "2.1.8",
```

Comment:
Do we still need `"node-sass"` if we're depending on `"postcss-node-sass"` seems like it'll be installed anyways